test/Transforms/set-spirv-capability.mlir

```diff
@@ -0,0 +1,41 @@
+// RUN: imex-opt --set-spirv-capablilities %s | FileCheck %s
+
+module attributes {gpu.container_module} {
+
+// CHECK: module attributes {gpu.container_module, spv.target_env = #spv.target_env<#spv.vce<v1.0, [Addresses, Float16Buffer, Int64, Int16, Int8, Kernel, Linkage, Vector16, GenericPointer, Groups, Float16, Float64, AtomicFloat32AddEXT, ExpectAssumeKHR], [SPV_EXT_shader_atomic_float_add, SPV_KHR_expect_assume]>, #spv.resource_limits<>>} {
```

Comment:
Any particular reason why this test is so big? Most of the code here is not checked, so why is it needed?
It might be worthwhile generally reducing the suggested tests. Minimal tests are easier to maintain and understand.

Reply:
changed